plugins/ca/qml/Keyboard_ca_email.qml

```diff
@@ -14,7 +14,7 @@
  * along with this program.  If not, see <http://www.gnu.org/licenses/>.
  */
 
-import QtQuick 2.4
+import QtQuick 2.9
```

Comment:
Is there any reason to use newer QtQuick versions when we aren't using the newer API?

Reply:
Isn't 2.9 a bit faster than 2.4?
I thought it was advisable to use the latest possible version...